### PR TITLE
[REGEDIT] *.rc: Strip useless WS_VISIBLE STYLE from dlgs

### PR DESCRIPTION
**Pull request by JoachimHenze**
360 matches, exactly 12 matches for each language, 30 languages.

This will not shrink the binary, it just shortens some long lines within the rc-sources.

JIRA issue: none

### CODE REVIEW

**Comment by whindsaks** [comment]
And this is safe because the code manually calls `ShowWindow`? `DialogBox` forces visible?

**Comment by HBelusca** [comment]
> And this is safe because the code manually calls `ShowWindow`? `DialogBox` forces visible?

https://learn.microsoft.com/en-us/windows/win32/dlgbox/about-dialog-boxes#modal-dialog-boxes
> ## Modal Dialog Boxes
>
> A modal dialog box should be a pop-up window having a window menu, a title bar, and a thick border; that is, the dialog box template should specify the WS_POPUP, WS_SYSMENU, WS_CAPTION, and DS_MODALFRAME styles. Although an application can designate the WS_VISIBLE style, the system always displays a modal dialog box regardless of whether the dialog box template specifies the WS_VISIBLE style. An application must not create a modal dialog box having the WS_CHILD style. A modal dialog box with this style disables itself, preventing any subsequent input from reaching the application.

_**HOWEVER:**_

https://learn.microsoft.com/en-us/windows/win32/dlgbox/about-dialog-boxes#modeless-dialog-boxes
> ## Modeless Dialog Boxes
>
> A modeless dialog box should be a pop-up window having a window menu, a title bar, and a thin border; that is, the dialog box template should specify the WS_POPUP, WS_CAPTION, WS_BORDER, and WS_SYSMENU styles. The system does not automatically display the dialog box unless the template specifies the WS_VISIBLE style.

File: base/applications/regedit/lang/bg-BG.rc [hunk 1]
```diff
@@ -643,7 +643,7 @@ END
 /* Dialogs */
 
 IDD_EXPORTRANGE DIALOGEX 50, 50, 370, 50
-STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS | WS_BORDER
+STYLE DS_SHELLFONT | DS_CONTROL | WS_CHILD | WS_CLIPSIBLINGS | WS_BORDER
```
Comment (HBelusca) [inline base/applications/regedit/lang/bg-BG.rc:646]: Please check that the Save-As dialog (when e.g. exporting a registry key) still correctly shows the extra thing.

Comment (JoachimHenze) [reply base/applications/regedit/lang/bg-BG.rc:646]: What do you mean with "the extra thing"?

Comment (HBelusca) [reply base/applications/regedit/lang/bg-BG.rc:646]: export_key
This :point_up:

Comment (JoachimHenze) [reply base/applications/regedit/lang/bg-BG.rc:646]: To me the export dlg looks good:
image
registry-export-dlg is modal.

Comment (JoachimHenze) [reply base/applications/regedit/lang/bg-BG.rc:646]: running our patched regedit on Windows XP SP3, does look also good:
image

Comment (HBelusca) [reply base/applications/regedit/lang/bg-BG.rc:646]: should be good then :+1:

**Review by HBelusca — APPROVED** [review]
All these IDD_* in regedit are opened with `DialogBox(Param)`, i.e. they are modal, so they will be made visible regardless of the `WS_VISIBLE` flag. So the PR looks OK.

**Review by whindsaks — APPROVED** [review]

**Pull request merged**